Copying.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
```

Comment:
<p>Maybe we can rename black box to "controls" or so? i find this line in the flow a bit distracting. Also, can we subscribt the 2 like?  $O_2$? :)--not in this comment it seems</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>